index.html

```diff
@@ -0,0 +1,350 @@
+          <figcaption>
+            <h4 class="portfolio-caption-header">Management</h4>
+            <p class="portfolio-category">UI design, brand &amp; logotypes</p>
+          </figcaption>
+        </figure>
+
+        <figure class="portfolio-item">
+          <a class="portfolio-link-hover" href=""><img class="portfolio-opacity" src="/img/korean.jpg" alt="Image of recipe app screen for korean bulgogi beef" title="Korean beef recipe screen"></a>
+          <figcaption>
+            <h4 class="portfolio-caption-header">Korean beef</h4>
+            <p class="portfolio-category">Webdesign, brand &amp; logotypes</p>
+          </figcaption>
+        </figure>
+
+        <figure class="portfolio-item">
+          <a class="portfolio-link-hover" href=""><img class="portfolio-opacity" src="/img/facebook-thumb.jpg" alt="Image of Facebook like and dislike thumb icons" title="Facebook thumbs"></a>
+          <figcaption>
+            <h4 class="portfolio-caption-header">Facebook thumbs</h4>
+            <p class="portfolio-category">Icons design, brand &amp; logotypes</p>
+          </figcaption>
+        </figure>
+
+      </article>
+      <article id="about" class="about">
+        <h2 class="about-header white-line">ABOUT</h2>
+        <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.</p>
+        <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.</p>
+        <section>
+          <h3 class="skills-header white-line">SKILLS</h3>
+          <ul class="skills">
+            <li>
+              <h4 class="progress-header">PHOTOSHOP</h4>
+              <progress value="77" max="100"></progress>
+            </li>
+            <li >
+              <h4 class="progress-header">HTML/CSS</h4>
+              <progress value="91" max="100"></progress>
+            </li>
+            <li>
+              <h4 class="progress-header">ILLUSTRATOR</h4>
+              <progress value="69" max="100"></progress>
+            </li>
+            <li>
+              <h4 class="progress-header">PHP</h4>
+              <progress value="55" max="100"></progress>
+            </li>
+            <li>
+              <h4 class="progress-header">WORDPRESS</h4>
+              <progress value="97.5" max="100"></progress>
+            </li>
+          </ul>
+        </section>
+        <aside>
+          <h3 class="join-us-header">Do you want to be one of Avengers? Join us!</h3>
+          <button class="contact-us-button">CONTACT US</button>
```

Comment:
Based on the flow of the mock-up, I'd guess that the desired behavior here is that when a user clicks on "Contact Us" they should be taken to the contact form. In that case, an anchor element would be appropriate here, with a `href` attribute of `#contact`.

Tangentially -- if you're going to use a `button`, please assign a `type` attribute (in this case, if you were opening up a modal dialog via JavaScript, for example, it would be `<button type="button"></button>`).